### ISSUE TITLE
mi6 build removes license from META6.json

### ISSUE THREAD

**Issue by kuerbis** [skaji/mi6#17]
Hello,

when I call `mi6 build`, the line `"license" : "Artistic-2.0",` is removed from META6.json.

**Comment by samcv** [comment]
@kuerbis is the project on Github? Do you have a LICENSE file? Or can you paste the META6.json so we can try and reproduce? Thank you!

**Comment by skaji** [comment]
@kuerbis Please update your mi6.

**Comment by kuerbis** [comment]
It is on github: https://github.com/kuerbis/Term-Choose-p6

I tried to update App::Mi6 with
`zef update`
`zef upgrade App::Mi6`
also with
`zef uninstall App::Mi6`
`zef install App::Mi6`
I don't know how to find out the module version in perl6.
But it still removes the license line.

**Comment by samcv** [comment]
It doesn't look like the META has a license field:
https://github.com/kuerbis/Term-Choose-p6/blob/master/META6.json

When I run it, it actually adds a license field.

```patch
 META6.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/META6.json b/META6.json
index 2404928..9f771f6 100644
--- a/META6.json
+++ b/META6.json
@@ -9,6 +9,7 @@
     "Terminal::WCWidth"
   ],
   "description" : "Choose items from a list interactively.",
+  "license" : "Artistic-2.0",
   "name" : "Term::Choose",
   "perl" : "6.c",
   "provides" : {
```

**Comment by samcv** [comment]
You could try deleting the folders in ~/.zef to delete the cache and make sure it downloads a new version when you reinstall it. That is what I would try next. Sometimes zef is overzealous with its caching.

**Comment by ugexe** [comment]
Note the cause of this is the lack of version. How zef's cache handles versions is how CUR handles versions, so you can *only* install these version * modules with --force when they are updated.

tl;dr - use a version in META6.json

**Comment by kuerbis** [comment]
I deleted the folders in ~/.zef and installed App::Mi6 with --force. Now it works fine: `mi6 build` doen't remove the license file any more. 
(`mi6 release` tells to install the module with `panda`)

**Comment by skaji** [comment]
Now mi6 specifies a version.
And you can even install it from CPAN http://www.cpan.org/authors/id/S/SK/SKAJI/Perl6/, i think:)
